api.py

```diff
@@ -24,9 +24,9 @@ def toDict(self):
         }
 
 
-    def fromDict(self):
-        # TODO
-        raise NotImplementedError
+    def fromDict(self, data: dict):
```

Comment:
class `JSONable` , `FileResult` , `TCResult` 모두 해당하는 변경사항:
```
@classmethod                       # Add classmethod decorator
def fromDict(cls, data: dict):     # Change param name self -> cls
    return cls(???)                # Call constructor via cls param instead of hardcoding class type
```

단, `JSONable`의 function body 는 `raise NotImplementedError` 그대로 유지